sounddevice.py

```diff
@@ -1877,9 +1877,19 @@ def input_underflow(self):
         This can only happen in full-duplex streams (including
         `playrec()`).
 
+        The bit may be set with ``input_underflow = True``, and
+        cleared with ``input_underflow = False``.
+
         """
         return self._hasflag(_lib.paInputUnderflow)
 
+    @input_underflow.setter
+    def input_underflow(self, new_value):
```

Comment:
I don't think the `new_` part of `new_value` is necessary. I would prefer `value`.

See also https://docs.python.org/3/library/functions.html#property.

Reply:
I've removed the "new_" part from each setter.